justfile

```diff
@@ -289,10 +289,19 @@ coordinator args="":
 
     cargo run --bin coordinator -- {{args}}
 
+run_maker_args := if os() == "linux" {
```

Comment:
Nice, alternatively you could use 

```
docker run -d --name maker ghcr.io/get10101/aristides/aristides:main regtest --orderbook http://172.17.0.1:8000
```

It seems that on Linux `172.17.0.1` is always `localhost`

That being said, can you update the CI file to use this api as well?

https://github.com/holzeis/10101/blob/d29baf6acdfa262aa0c7e1b84ecc581e036f9b8b/.github/workflows/ci.yml#L276